libs/feature/dataviz/src/lib/service/data.service.ts

```diff
@@ -83,12 +84,26 @@ export class DataService {
                 outputCrs: 'EPSG:4326',
               })
             : null,
+          gml:
+            featureType.outputFormats.find((f) =>
+              f.toLowerCase().includes('gml')
+            ) && featureType.otherCrs?.includes('EPSG:4326')
```

Comment:
Why do you use `4326` ? When are data reprojected ?

Reply:
In order to check if 4326 is available and so, be readable by the ol/format/WFS class. The condition would be better : 
```
... (featureType.defaultCrs === 'EPSG:4326' || featureType.otherCrs?.includes('EPSG:4326') )
```

We can even have a look if EPSG:3857 is available too. 

As it is to cover some olds map servers, maybe we can rely only on 4326 ? 
What do you think of it ?

Reply:
Yes your check might be better, it's easy to add a test that confirms that. I think that 4326 should be supported by any map server though..

Anyway, my question was more "_If you get `4326`, and the geojson request might returns `4326` as well, where in the code, in the end, the geojson is projected back to `3857` ?_"

The answer is here https://github.com/geonetwork/geonetwork-ui/blob/main/libs/feature/map/src/lib/map-context/map-context.service.ts#L111
Which transforms the geojson from `4326` to `3857` to be displayed on the map, which means we must request the GML in `4326`. Note that for the table and the dataviz, the projection is useless as the geometry is not used.

So it's all good.

Reply:
Alright, I'm updating this and fix CI.

Reply:
@fgravin  if it's ok for you, I can merge it :)